pycaret/internal/plots/time_series.py

```diff
@@ -1025,3 +1025,89 @@ def plot_time_series_decomposition(
 
     return fig, return_data_dict
 
+
+def plot_multiple_predictions_with_confidence(
+        data: pd.Series,
+        predictions: List,
```

Comment:
This should be a Union[pd.Series,  List[pd.Series]].

It should be able to accept single series or a List of series.